engines/wasmer/wasmer.go

```diff
@@ -126,6 +144,11 @@ func (e *engine) New(_ context.Context, host wapc.HostCallHandler, guest []byte,
 	}, nil
 }
 
+// Unwrap allows access to wasmer-specific features.
+func (m *Module) Unwrap() *wasmer.Module {
```

Comment:
an alternative to doing like this is having `Unwrap interface{}` on the top-level type, though the end user will need to cast regardless. (ex here they cast first then don't cast on Unwrap, but if on the top-level type, they don't cast first, but cast on Unwrap)